pkg/disruptors/pod.go

```diff
@@ -108,7 +115,12 @@ func (d *podDisruptor) InjectHTTPFaults(
 ) error {
 	cmd := buildHTTPFaultCmd(fault, duration, options)
 
-	err := d.controller.ExecCommand(ctx, cmd)
+	err := d.validatePort(ctx, fault.Port)
+	if err != nil {
+		return fmt.Errorf("verify fault port: %w", err)
```

Comment:
I think this error description could be reworded as "port validation error: " or something similar for clarity